adjust_dataset.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
+import argparse
+
+HAPPINESS = 3
+
+# Alternative for users who don't pass arguments from command-line, that alternative will be deleted in future version
+path_input_file = "fer2013.csv"
+path_output_file = 'smile_warrior_dataset.csv'
+
+# Set that flag to 1 if you want to pass paths for csv files as a command-line arguments
```

Comment:
This looks like an overkill, cli argument will suffice. Keep It Simple Stupid. And for future - use booleans instead of 0/1.

Reply:
Ok